<?xml version="1.0"?>
<robot name="calibration_mire">
  <handle name="left" clearance="0.04">
    <!-- 180° around z and then 90° around x' -->
    <position>  0.21 0 0 0 0 -0.7071067811865476 0.7071067811865476 </position>
    <link name="base_link"/>
  </handle>
  <handle name="right" clearance="0.04">
    <!-- 90° around x -->
    <position>  -0.21 0 0 0.7071067811865476 0.7071067811865476 0 0 </position>
    <link name="base_link"/>
  </handle>
</robot>
